packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/sign-the-csr/index.md

```diff
@@ -10,6 +10,6 @@ The CA that you choose provides instructions on how to upload the CSR that you g
 
 The CSR is generated in Base64 DER format. If your process requires a different format, convert it using OpenSSL or a third-party decoder. Free, third-party decoders are readily available.
 
-After you update the key credential, your users can't access the SAML app until you upload the new certificate to the Service Provider (SP).
+> **Caution** After you update the key credential, your users can't access the SAML app or the Identity Provider until you upload the new certificate to the Service Provider (SP).
```

Comment:
This content needs to be removed from `Sign the CSR` step and added to the step `Update the key credential`.

Reply:
<img width="966" alt="Screen Shot 2020-07-01 at 3 09 24 PM" src="https://user-images.githubusercontent.com/45509385/86291922-096b8880-bbad-11ea-8101-050c2250a713.png">
moved it